### ISSUE TITLE
ToUpdate

### ISSUE THREAD

**Issue by caiyinqiong** [caiyinqiong/Semantic-Retrieval-Models#6]
Pre-training：
1. Hyperlink-induced Pre-training for Passage Retrieval in Open-domain Question Answering（ACL2022）
2. RetroMAE v2: Duplex Masked Auto-Encoder For Pre-Training Retrieval-Oriented Language Models(Arxiv, 2022)

**Comment by xiahaoyun** [comment]
Multi-vector Representation：
1. Rethinking the Role of Token Retrieval in Multi-Vector Retrieval ([arXiv](https://arxiv.org/pdf/2304.01982.pdf) ,[twitter](https://twitter.com/leejnhk/status/1643632578824396805?s=20))  
2. SparseEmbed: Learning Sparse Lexical Representations with Contextual Embeddings for Retrieval（SIGIR2023）

**Comment by caiyinqiong** [comment]
end-to-end training：
1. Retrieval as Attention: End-to-end Learning of Retrieval and Reading within a Single Transformer

**Comment by caiyinqiong** [comment]
few-shot:
1. Rethinking Dense Retrieval’s Few-Shot Ability

**Comment by caiyinqiong** [comment]
Retrieval-Augment LLM:
1. REPLUG: Retrieval-Augmented Black-Box Language Models  
2. Shall We Pretrain Autoregressive Language Models with Retrieval? A Comprehensive Study

**Comment by caiyinqiong** [comment]
LLM for Retrieval:
1. Precise Zero-Shot Dense Retrieval without Relevance Labels

**Comment by caiyinqiong** [comment]
可解释性：
1. Explain like I am BM25: Interpreting a Dense Model’s Ranked-List with a Sparse Approximation
